### PR TITLE
Bump revision and update changelog

### PR DESCRIPTION
**Pull request by hassount**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+- Loop filter flow optimization
+
+Encoder and Decoder
+- Encoder-Decoder-Common folder structure improvements
+
+Build and Testing
+- Bug fixes
+- Improve CI
+- Improve Unit Test Coverage
+- Address C vs asm mismatches
```
Comment (EwoutH) [inline CHANGELOG.md:37]:
```suggestion
- Address C vs asm mismatches
- Support C only builds (for platforms other than x86)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+
```
Comment (EwoutH) [inline CHANGELOG.md:38]:
```suggestion
- Support C only builds (for platforms other than x86)

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## [0.8.2] - 2019-04-18
```
Comment (jryzkns) [inline CHANGELOG.md:3]: Do you perhaps mean "2020-04-18"?

Comment (hassount) [reply CHANGELOG.md:3]: yes thank you @jryzkns it seems that we needed to update 0.8.1 too.

**Comment by hassount** [comment]
@joelsole, is this good to go ?

**Comment by tianjunwork** [comment]
Also root CMakelist.
project(svt-av1 VERSION 0.8.1
    LANGUAGES C CXX ASM_NASM)

**Pull request merged**